README.md

```diff
@@ -1,27 +1,24 @@
-# Ardupilot Gazebo plugin 
+# ArduPilot Ignition Gazebo plugin
+This is ArduPilot official plugin for Ignition Gazebo.
+It replaces the old Gazebo plugin to bring support of the next gen Gazebo simulator.
+It also brings support for more features :
+- more flexible data exchange between SITL and Ignition with JSON data sharing.
+- more sensors supported.
+- true Simulation lockstepping. It is now possible to use GDB to stop the Ignition time for debugging.
+- Better 3D rendering
+
+## Disclaimer :
+The plugin is currently working, but we are working into bringing support for more feature and refine the API.
 
 ## Requirements :
-Native Ubuntu able to run full 3D graphics.
-Gazebo version 8.x or greater
-The dev branch will works on gazebo >= 8.x  
-For Gazebo 7 use branch gazebo7
-
-## Disclamer : 
-This is a playground until I get some time to push the correct patch to gazebo master (I got hard time to work with mercurial..)!  
-So you can expect things to not be up-to-date.  
-This assume that your are using Ubuntu 16.04 or Ubuntu 18.04
+Ubuntu able to run full 3D graphics: Native, VM, WSL, Docker.
+Ignition Gazebo version 5.x (Ignition Edifice) or greater. See https://ignitionrobotics.org/docs/edifice/install_ubuntu
 
 ## Usage :
```

Comment:
This section could be 'Install' as it covers installing prerequisites and the plugin. The 'Usage' section could cover how to launch gazebo with one of the example models and configure SITL. 

There is more information on the project wiki https://github.com/ArduPilot/ardupilot_gazebo/wiki that can be linked to from the Usage section.